visual_behavior/data_access/loading.py

```diff
@@ -428,7 +428,7 @@ def _get_events(self):
 
                 meta_string = self.metadata_string
                 if 'MESO' in meta_string:  # change time scale for filtering events for meso
-                    scale = 2/3
+                    scale = 2 / 3
```

Comment:
What is the exact ratio of the sampling rates?

Reply:
this is just what Peter L told me